thonny/running.py

```diff
@@ -916,8 +974,42 @@ def _get_environment(self):
             env["THONNY_DEBUG"] = "1"
         elif "THONNY_DEBUG" in env:
             del env["THONNY_DEBUG"]
+
+        add_env_file = self._load_env_file()
+
+        chained_env = collections.ChainMap(add_env_file, env)
+        chained_env = dict(chained_env)
+
+        # un-set keys
+        env = EnvFile()
+        env.set_env(chained_env)
+        env.remove_unused()
```

Comment:
I see you are treating the empty value as signal to remove the environment variable. Is this standard behavior? Why not simply keep the variable with empty value?

Reply:
a good point!

but how then to support not existence when falling back to the programmers default with `os.environ.get("somekey", defaultval)` is the wanted behaviour ?

how about adding an additional hard test against double and single quoted strings to get this feature for empty strings.

or how about to stay with that implementation and wait if issues are created asking for such a feature, where someone has to point out why it is required. thinking here of a step-by-step process to add more features. also because ".env" file settings and pythons configparser functionallity might overlap here.

Reply:
I think this should work just like in other dotenv implementations (in case there is no specification which clears this issue).

Reply:
there might be a specification as implementation (aka. living standard by dotenv module itself)
i keep up with having bash var behaviour in mind. 
in this case treated as empty string.